app/screens/settings/display_settings/display_settings.js

```diff
@@ -83,10 +114,11 @@ export default class DisplaySettings extends PureComponent {
                         iconName='ios-time'
                         iconType='ion'
                         onPress={this.goToClockDisplaySettings}
-                        separator={false}
+                        separator={true}
```

Comment:
this should be `true` only if the server has the timezone feature available

Reply:
no need anymore, the timezone feature will be set by default